crates/cargo-wdk/README.md

```diff
@@ -0,0 +1,101 @@
+# cargo-wdk
+
+A Command-Line Interface (CLI) utility to create and build Windows driver projects written in Rust. 
+`cargo-wdk` is a cargo extension (plugin) with "cargo-like" commands to
+- create new projects from scratch 
+- build new or existing projects
+
+**NOTE**: `cargo-wdk` utility is designed to work with projects that depend on the Windows Driver Kit (WDK) and the "windows-drivers-rs" platform
+
+## Installation
+
+To install cargo-wdk, you need to have Rust and Cargo installed on your system. 
+You can install Rust and Cargo by following the instructions on the [Rust website](https://www.rust-lang.org/tools/install).
+
+Once you have Rust and Cargo installed, you can install cargo-wdk using the cargo install command in one of the following ways:
+
+### Install by cloning the source
+    - Clone the windows-drivers-rs repository.
+    - Navigate to crates\cargo-wdk directory.
+    - Run the following command:
+        ```pwsh
+        cargo install --path .
+        ```
+
+### Install by specifying Git repository
+    - Run the following command:
+        ```pwsh
+        cargo install --git https://github.com/microsoft/windows-drivers-rs.git --bin cargo-wdk
+        ```
+
+The install command compiles the `cargo-wdk` binary and copies the compiled binary to Cargo's bin directory (%USERPROFILE%.cargo\bin).
```

Comment:
No need for this as it is an internal implementation detail of Cargo. The user does not need to know it.